wherehows-web/app/components/datasets/health/score-table.ts

```diff
@@ -76,11 +76,11 @@ export default class DatasetsHealthScoreTable extends Component {
   headers = computed('dropdownOptions', function(this: DatasetsHealthScoreTable): Array<IHealthTableHeader> {
     const { dropdownOptions, onDropdownSelect } = getProperties(this, 'dropdownOptions', 'onDropdownSelect');
 
-    return ['Category', 'Description', 'Score', 'Severity'].map(header => ({
-      label: header,
+    return (['category', 'description', 'score', 'severity'] as Array<keyof typeof HealthDataFields>).map(header => ({
```

Comment:
```ts
Object.keys(HealthDataFields).map((header: string) => ({ ... }))
```

Reply:
Ah, was going to say that I was concerned about the order of Object.keys() returned array, but apparently that's not really an issue anymore in ES2015 onwards.